src/main/clm_driver.F90

```diff
@@ -1223,8 +1224,22 @@ subroutine clm_drv(doalb, nextsw_cday, declinp1, declin, rstwr, nlend, rdate, ro
        ! ============================================================================
        ! Determine albedos for next time step
        ! ============================================================================
-
-       if (doalb) then
+       
+       if (.not.doalb) then
+
+
+          if(use_fates) then
+             ! During branch runs and non continue_run restarts, the doalb flag
```

Comment:
Hi @rgknox

I mentioned this last week. And I don't want discussion here to slow down bringing this tag in as is. 

But, this is the part that I wondered about if it really needs to be done all the time. My suspicion is that you may only need to do it under special cases after a branch restart. nextsw_cday isn't going to change until doalb is true, so I think the zenith angles shouldn't need to be updated here, except under those special circumstances.

So after the tag, is made I'd like to discuss this a bit at some point. I think it would be good to restrict this logic to just the cases when it's really needed.

Let me know when a good day/time to chat about this would be. Thanks.

Reply:
In favor of merging this PR, I am resolving this conversation at the risk of any follow-up getting forgotten...